### PR TITLE
Personality: allow lua, cs and pas files

### PR DESCRIPTION
**Pull request by tobitege**
In Discussions files with extension "lua", "cs" or "pas" were rejected.  
Fixed by adding 3 file extensions to personality events, also removed 1 duplicate "js" from the list.

### CODE REVIEW

File: lollms/server/events/lollms_personality_events.py [hunk 1]
```diff
@@ -31,27 +31,27 @@
...
+            trace_exception(ex)
 
     import os
     import imghdr
     import mimetypes
 
     ALLOWED_EXTENSIONS = {
         'txt', 'csv', 'py', 'html', 'js', 'jpg', 'jpeg', 'png', 'gif', 'bmp', 'tiff', 'ico', 'svg', 'mp4', 'mp3', 'avi', 'mov',
-        'doc', 'docx', 'ppt', 'pptx', 'xls', 'xlsx', 'pdf', 'js', "md", "json"
+        'doc', 'docx', 'ppt', 'pptx', 'xls', 'xlsx', 'pdf', 'md', 'json', 'lua', 'cs', 'pas', 'ts'
```
Comment (ParisNeo) [inline lollms/server/events/lollms_personality_events.py]: Can you add js here?

**Comment by tobitege** [comment]
2nd commit added more extensions. This should be ready.

**Review by ParisNeo — APPROVED** [review]
legit

File: lollms/server/events/lollms_personality_events.py (hunk 1 above)
Comment (tobitege) [reply lollms/server/events/lollms_personality_events.py]: js is already in line 53, I removed a duplicate :)

**Pull request merged**